Automation_Test_Framework/TestSetData/TestSetDataAsset.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Core_Framework.DriverCore;
+using OpenQA.Selenium;
+using TheRookiesApp.DAO;
+
+namespace TheRookiesApp.TestSetData;
+
+public class TestSetDataAsset : WebDriverBase
+{
+    public TestSetDataAsset(IWebDriver? driver) : base(driver)
```

Comment:
Why do we need to inherit the WebDriverBase and define driver here?

Reply:
Done!